app/util/networks/handleNetworkSwitch.ts

```diff
@@ -12,6 +12,7 @@ import { store } from '../../store';
 /**
  * Switch to the given chain ID.
  *
+ * @param switchToChainId - This chain Id have a decimal format, usually provinient from deeplinks
```

Comment:
Typo here, maybe this was what was meant?

```suggestion
 * @param switchToChainId - This chain ID has a decimal format, usually provided from deeplinks
```

Reply:
Thanks for this catch!! Yess!!